package.json

```diff
@@ -241,7 +241,8 @@
     "inquirer": "7.3.3",
     "json-bigint": "1.0.0",
     "lodash": "4.17.21",
-    "lodash-es": "4.17.21",
+    "lodash-es": "4.17.15",
+    "matomo-tracker": "^2.2.4",
```

Comment:
Should we set the this package to a fixed version?

Reply:
Good catch, fixed